timeseries/setup.py

```diff
@@ -32,6 +32,7 @@
     "torch>=1.9,<1.14",
     "pytorch-lightning>=1.7.4,<1.9.0",
     "networkx",
+    "statsforecast==1.4.0",
```

Comment:
Can we make `1.4.0` the lower bound?

Reply:
This way we are protecting ourselves from potentially breaking changes / regressions caused by newer versions of the dependencies. Both things already happened to us a few times (caused by minor releases of sktime & GluonTS), so I would rather be extra cautious here.